interfaces/builtin/cifs_mount.go

```diff
@@ -19,33 +19,33 @@
 
 package builtin
 
-const cifsMountControlSummary = `allows mounting and unmounting CIFS filesystems`
+const cifsMountSummary = `allows mounting and unmounting CIFS filesystems`
 
-const cifsMountControlBaseDeclarationSlots = `
-  cifs-mount-control:
+const cifsMountBaseDeclarationSlots = `
+  cifs-mount:
     allow-installation:
       slot-snap-type:
         - core
     deny-auto-connection: true
 `
 
-const cifsMountControlConnectedPlugSecComp = `
+const cifsMountConnectedPlugSecComp = `
 # Description: Allow mount and umount syscall access.
 
 mount
 umount
 umount2
 `
 
-const cifsMountControlConnectedPlugAppArmor = `
+const cifsMountConnectedPlugAppArmor = `
 # Description: Allow mounting and unmounting CIFS filesystems.
 
 # Required for mounts and unmounts
 capability sys_admin,
 
 # Allow mounts to our snap-specific writable directories
-mount fstype=cifs ** -> /var/snap/@{SNAP_NAME}/@{SNAP_REVISION}/{,**},
-mount fstype=cifs ** -> /var/snap/@{SNAP_NAME}/common/{,**},
+mount fstype=cifs //** -> /var/snap/@{SNAP_NAME}/@{SNAP_REVISION}/{,**},
+mount fstype=cifs //** -> /var/snap/@{SNAP_NAME}/common/{,**},
```

Comment:
Can you comment that this actually works for you?

Reply:
yes, I tested it with my minimal test snap and everything seemed to work fine:

https://github.com/datenhahn/devtools-cifs-mount-control